amivapi/tests/events/test_emails.py

```diff
@@ -229,3 +229,39 @@ def test_no_nones_in_emails(self):
         mail = self.app.test_mails[1]
         for field in mail.values():
             self.assertTrue('None' not in field)
+
+    def test_signup_added_by_admin_accepted_emails(self):
+        """Test that an accepted email is sent when an admin has added a
+        signup manually."""
+        event = self.new_object('events', spots=1, selection_strategy='fcfs',
+                                allow_email_signup=True)
+
+        user = self.new_object('users')
+
+        self.api.post('/eventsignups', data={
+            'user': str(user['_id']),
+            'event': str(event['_id']),
+            'accepted': True
+        }, token=self.get_root_token(), status_code=201).json
+
+        mail = self.app.test_mails[0]
+        for field in mail.values():
+            self.assertTrue('None' not in field)
+
+        self.assertTrue('accepted' in mail['subject'])
+
+        self.api.post('/eventsignups', data={
+            'email': 'a@example.com',
+            'event': str(event['_id']),
+            'accepted': True
+        }, token=self.get_root_token(), status_code=201).json
+
+        # accepted email and confirmation email should be sent.
+        self.assertIs(len(self.app.test_mails), 3)
+        for mail in self.app.test_mails[1:]:
+            for field in mail.values():
+                self.assertTrue('None' not in field)
+
+            self.assertTrue(
+                ('accepted' in mail['subject']) or
+                ('confirm_email' in mail['text']))
```

Comment:
Arg, a bit the same issue as above. These assertions seem super fishy. I guess it's because of the way the test is set up? Maybe we can add a comment to explain why asserting this means that everything works?